src/main/resources/application.yml

```diff
@@ -0,0 +1,44 @@
+nfo:
+  app:
+    fullname: Hocs Search Service
+    name: ${name:hocs-search-service}
+    version: ${version:0.0.1}
+
+server:
+  port: 8088
+  shutdown: graceful
+
+spring:
+  main:
+    banner-mode: off
+
+elastic:
+  host: localhost
+  port: 4566
```

Comment:
We use AWS OpenSearch, could probably condense this under that set of properties?